openlibrary/templates/subjects.html

```diff
@@ -19,7 +19,7 @@ <h1 class="inline">
 
     <form action="/search" class="olform pagesearchbox">
       <input type="hidden" name="${page.subject_type}_facet" value="$page.name" />
-      <input type="text" name="q" id="searchSubjects" size="100" class="larger" value="" />
+      <input type="text" name="q" aria-label="Search" id="searchSubjects" size="100" class="larger" value="" />
```

Comment:
Here we definitely need a label. This implementation will definitely work. I'd suggest, at least, updating the value to read something like "Search Subjects".

While it would be a visually breaking change, if we could, it'd be nice to use static HTML `<label>` elements rather than ARIA - though, this would add some more clutter to an already cluttered page.

Reply:
This search bar is for Individual Subject page which search for books related to the subject so should I keep it "Search" same as header search bar and change the other one to "Search Subject" (the one on Subject Page)

Reply:
I think we want to make it clear to users that this search field is different that the main site search - so using something like "Subjects Search" (or something else, I don't have a huge preference here) - would help to let users know the page they'll be sent to is different than the default search page.

The other thing about using `aria-label` attributes is that we should be getting this as a string from the python variables so that we can internationalize the text. That's already an issue in the publisher search templates, but if we can find a way to get that working in this example, we can hopefully update the language in those labels to!

Reply:
Actually, we are sending users to the search page.
and on the search page we are using the following code:
```
<label for="q" class="hidden">$_('Keywords')</label>
<input type="text" name="q" id="q" value="$q_param" size="100"/>
<button type="submit" class="larger" id="searchsubmit">$_("Search")</button>      
```

Reply:
I think we should go ahead with:
```
<label for="q" class="shift">$_('Keywords')</label>
<input type="text" name="q" id="q" value="$q_param" size="100"/>
<button type="submit" class="larger" id="searchsubmit">$_("Search")</button>
```

Reply:
@bpmcneilly any feedback about proceeding w/ @Yashs911's recommended direction?

Reply:
@Yashs911 - Sorry for the delay here & my confusion about these different search functions. I agree with the direction you were suggesting!